manager/integration/negative_tests/lib/Operations.py

```diff
@@ -0,0 +1,62 @@
+import socket
+import time
+
+from kubernetes import config as k8sconfig
+from common import get_mgr_ips, get_client, PORT, Gi, create_volume as _create_volume, get_self_host_id, prepare_host_disk, \
```

Comment:
Suggest splitting the imports for readability.

Reply:
most of functions in common.py use longhorn python REST API client, but using longhorn custom resources is preferred in the future. Probably will not reuse/import functions in common.py.

Reply:
Suggest having the Longhorn HTTP API client as an option. This will be useful to test user-facing UI behaviors.